tools/nnicmd/launcher.py

```diff
@@ -205,8 +205,9 @@ def set_experiment(experiment_config, mode, port, config_file_name):
         return response
     else:
         _, stderr_full_path = get_log_path(config_file_name)
-        with open(stderr_full_path, 'a+') as fout:
-            fout.write(json.dumps(json.loads(response.text), indent=4, sort_keys=True, separators=(',', ':')))
+        if response:
```

Comment:
if this is true, does this mean response must have .text?

Reply:
yes.